pkg/cli/admin/upgrade/recommend/recommend.go

```diff
@@ -63,8 +63,8 @@ func New(f kcmdutil.Factory, streams genericiooptions.IOStreams) *cobra.Command
 type options struct {
 	genericiooptions.IOStreams
 
-	mockData              mockData
-	IncludeNotRecommended bool
+	mockData             mockData
+	ShowOutdatedReleases bool
```

Comment:
nit: can be private

Reply:
I've pushed 46d8a1c096dc -> 07d7b03745e5b3, which pivots `ShowOutdatedReleases` to 
the private `showOutdatedReleases`.